examples/deployment-chain/simple/simple-cloudrun/app.pipecd.yaml

```diff
@@ -0,0 +1,8 @@
+# Quick sync by rolling out the new version and switching all traffic to it.
+apiVersion: pipecd.dev/v1beta1
+kind: CloudRunApp
+spec:
+  name: inchain-simple-cloudrun
```

Comment:
To be unified with other examples, how about using the same name for directory and application name?

e.g. `chain-simple-cloudrun`

Reply:
Nice catch, addressed by https://github.com/pipe-cd/pipe/pull/2993/commits/11735b481165704333086f9dacc97912a6e99400 🙏